app/lib/dags/experimental.json.erb

```diff
@@ -0,0 +1,32 @@
+{
+  "output_dir_s3":
+    "s3://<%= @attribute_dict[:bucket] %>/samples/<%= @project_id %>/<%= @sample_id %>/expt",
+  "targets": {
+    <% if @attribute_dict[:fastq2] %>
+      "fastqs": ["<%= @attribute_dict[:fastq1] %>", "<%= @attribute_dict[:fastq2] %>"],
+      "srst2_out": ["output.log", "output__genes__ARGannot_r2__results.txt", "output__fullgenes__ARGannot_r2__results.txt", "amr_processed_results.csv", "amr_summary_results.csv"]
```

Comment:
same comment about deduplicating by moving the `srst2_out` line after the if-else block